deploy/Deploy-ssh.yml

```
@@ -0,0 +1,38 @@
+            echo "Starting deployment..."
+
+            # Check if the Git repository already exists, if not, clone it
+            if [ ! -d ".git" ]; then
+              echo "Git repository not found. Cloning repository..."
+              git clone ${{ secrets.REPO_URL }}
+            else
+              echo "Git repository found. Pulling the latest changes..."
+              cd Docker
+              git pull origin main
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)